tests/main/snap-env/task.yaml

```diff
@@ -1,12 +1,15 @@
 summary: inspect all the set environment variables prefixed with SNAP_ and XDG_
+
 prepare: |
-    snap pack $TESTSLIB/snaps/test-snapd-tools
-    snap install --dangerous test-snapd-tools_1.0_all.snap
+    snap install test-snapd-tools
```

Comment:
This could do a `install_local`, right? I mean, the change also changes behaviour, is there a reason for this? (I don't mind I just want to understand the rational).